### DISCUSSION TITLE
FINALLY updating from 1.3something...to 2.1...is that the latest?...and maybe last version?

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#351] [General]
Hello! For ages I have been meaning to update Adam Image from 1.3.1 to the latest version...which is 2.1?...is that right?...and seems like maybe it will be the final version? Wondering if these things will now be straightforward?

Quake 2...maybe there is a way with the version I have, but just dropping the opk files in what seemed like the right place on the external SD card didn't seem to work.

Amiga emulation also. No luck so far. I think I have all the roms in the right place and the necessary bios files...but nothing works.

VERY excited to just discover that I have both Discworld games in with the ScummVM emulating games. Read sooo many Discworld books as a youth. Not quite decided how good point and click adventure games will feel on the 350m. Actually quite nice, those low res graphics on a small screen...as I have found with DOOM and DOOM 2.

Very enamoured with my Retroid 3+, but unlike my 2+ (that barely see's any action these days) my 350m still gets use, as I haven't figured out how to get DOOM on to it yet (my 3+), or Scummvm for example. I expect both can be put on there pretty easily...in fact a Scummvm app is just waiting to be added...BUT I might deliberately keep a few things only on my 350m...DOOMs, Quake(s), Amiga and Scummvm...maybe...oh also have Pico8 currently only on 350m...I haven't found a LCD filter/shader for Gameboy and GBA that works as well as that with in Adam Image...GBA defaulting to a stand alone emulator...ReGBA I think.

Ha! I might need to send this as a separate question...but anyone know how to save the "Keep Aspect Ratio" setting in RetroArch for Gameboy. I don't want it stretched to 4:3, but it always reverts back to this. From the main menu I go in to Settings/Configuration and Configuration File...Save Current Configuration, Save New Configuration...also from the Quick Menu, Overrides/Save Core Overrides and Save Game Overrides...but when I exit out of the game and then open it up again, back to stretched to 4:3. What am I doing wrong?

**Comment by zugurudumba** [comment]
I remember the devs mentioning another possible update some time ago, but I'm not sure it will ever be released. Meanwhile, I manually updated RetroArch from here https://buildbot.libretro.com/stable/1.15.0/dingux/mips32-odbeta/RetroArch.7z
